### PR TITLE
Updates to the Connection Classes

### PR DESCRIPTION
**Pull request by renatonascalves**
Adding minor updates to the connection classes.

### CODE REVIEW

File: .phpcs.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 	<!-- Rules: Check PHP version compatibility -->
 	<!-- https://github.com/PHPCompatibility/PHPCompatibility#sniffing-your-code-for-compatibility-with-specific-php-versions -->
-	<config name="testVersion" value="5.6-"/>
+	<config name="testVersion" value="7.0-"/>
```
Comment (kidunot89) [inline .phpcs.xml]: WPGraphQL has officially deprecated PHP 5.6, despite PHP 5.6 support being broken atm. So it's probably best to leave it as `5.6` for now.

File: README.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ Contributors: kidunot89, ranaaterning, jasonbahl, saleebm
 Tags: GraphQL, WooCommerce, WPGraphQL
 Requires at least: 4.9
 Tested up to: 5.2
-Requires PHP: 5.6
+Requires PHP: 7.0
```
Comment (kidunot89) [inline README.txt]: same as above with the version, and add your name to contributors list.

File: includes/connection/class-cart-items.php [hunk 3]
```diff
@@ -26,63 +29,66 @@ public static function register_connections() {
...
-					'resolve'     => function( $source ) {
-						if ( empty( $source['edges'] ) ) {
-							return 0;
-						}
+							$count = 0;
+							foreach ( $items as $item ) {
+								$count += $item['quantity'];
+							}
+
+							return $count;
```
Comment (kidunot89) [inline includes/connection/class-cart-items.php]:
```suggestion
							return array_sum( array_column( $items, 'quantity' ) );
```

File: includes/connection/class-wc-terms.php [hunk 4]
```diff
@@ -10,29 +10,33 @@
...
 namespace WPGraphQL\WooCommerce\Connection;
 
+use GraphQL\Type\Definition\ResolveInfo;
 use GraphQL\Error\UserError;
+use WPGraphQL\AppContext;
 use WPGraphQL\Connection\TermObjects;
+use WPGraphQL\Data\DataSource;
+use WPGraphQL\Data\Connection\TermObjectConnectionResolver;
+use WPGraphQL;
+use WP_GraphQL_WooCommerce;
```
Comment (kidunot89) [inline includes/connection/class-wc-terms.php]: There's little point in a `use` statement for a class that is only using called directly once.

Comment (renatonascalves) [reply includes/connection/class-wc-terms.php]: Hahah! Let's agree to disagree. But one point for it is visibility.

One can see all the classes being used in a file without the need to look/scan the file. Besides, it is such a small change that it doesn't hurt.

Happy to revert it if you feel strongly about it. :)

File: includes/connection/class-cart-items.php (hunk 3 above)
Comment (kidunot89) [reply includes/connection/class-cart-items.php]: This is cleaner

Comment (renatonascalves) [reply includes/connection/class-cart-items.php]: Nice! Tks!

**Comment by renatonascalves** [comment]
@kidunot89 Could you take another look? I added your code review suggestions. :)

**Pull request merged**